www/includes/easyparliament/templates/html/mp/_person_navigation.php

```diff
@@ -50,7 +50,7 @@
         <?php if (array_key_exists('letters_signed', $memberships) || array_key_exists('edms_signed', $memberships) || array_key_exists('topics_of_interest', $memberships) || array_key_exists('eu_stance', $memberships)): ?>
             <li <?php if ($pagetype == "signatures"): ?>class="active"<?php endif; ?>>
                 <a href="<?= $member_url ?>/signatures" class="person-navigation--subpage-heading">
-                    <h2>✍️ <?= gettext('Signatures') ?></h2>
+                    <h2>✍️ <?= gettext('Signatures') ?>🆕</h2>
```

Comment:
I think these need some sort of spacing as they are a bit jammed against the text:

<img width="238" height="57" alt="Screenshot 2025-09-15 at 14 23 13" src="https://github.com/user-attachments/assets/4480e9ae-fd80-4d25-8bfc-6cf8ae22827c" />

Reply:
I'll actually just roll this back for now - and can re-add when we do the launch.